src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -469,22 +550,61 @@ abstract class AbstractCheckTask(
     private fun Observable<StreamContainer>.continueObserve(sessionKey: SessionKey, sequence: Long): Observable<Message> =
         filter { streamContainer -> streamContainer.sessionKey == sessionKey }
             .flatMap(StreamContainer::bufferedMessages)
-            .filter { message -> message.metadata.id.sequence > sequence }
+            .filter { message ->
+                if (message.metadata.id.sequence == sequence || sequence == DEFAULT_SEQUENCE) {
+                    bufferContainsStartMessage = true
```

Comment:
I think we can set `bufferContainsStartMessage` to `true` if `sequence == DEFAULT_SEQUENCE` right in the begin method. We can remove this condition from here then